src/Header/HeaderWrap.php

```diff
@@ -90,7 +90,7 @@ protected static function wrapStructuredHeader($value, StructuredInterface $head
      */
     public static function mimeEncodeValue($value, $encoding, $lineLength = 998)
     {
-        return Mime::encodeQuotedPrintableHeader($value, $encoding, $lineLength, Headers::EOL);
+        return Mime::encodeBase64Header($value, $encoding, $lineLength, Headers::EOL);
```

Comment:
It's a BC break. It needs to be removed at this point, and a really, really strong case needs to be made for changing it.